pros/cli/upload.py

```diff
@@ -21,7 +21,7 @@ def upload_cli():
 @click.option('-q', '--quirk', type=int, default=0)
 @click.option('--name', 'remote_name', type=str, default=None, required=False, help='Remote program name',
               cls=PROSOption, group='V5 Options')
-@click.option('--slot', default=1, show_default=True, type=click.IntRange(min=1, max=8), help='Program slot on the GUI',
+@click.option('--slot', default=None, type=click.IntRange(min=1, max=8), help='Program slot on the GUI',
```

Comment:
I assume click is okay with the default value not matching the type specified?

Reply:
It looks fine, it compiles and runs just fine. The `--help` option looks like this.
`--slot INTEGER RANGE                 Program slot on the GUI`